plasmapy/formulary/relativity.py

```diff
@@ -88,3 +88,54 @@ def Lorentz_factor(V: u.m / u.s):
             gamma = ((1 - (V / c) ** 2) ** -0.5).value
 
     return gamma
+
+
+@validate_quantities(V={"can_be_negative": True}, m={"can_be_negative": False})
+def relativistic_energy(V: u.m / u.s, m: u.kg):
+    '''
+    Return the relativistic energy of a particle.
+
+    Parameters
+    ----------
+    V : ~astropy.units.Quantity
+        The velocity in units convertible to meters per second.
+
+    m : ~astropy.units.Quantity
+        The mass in units convertible to kilograms.
+
+    Returns
+    -------
+    E : float or ~numpy.ndarray
+        The relativitstic energy of the particle.
+
+    Raises
+    ------
+    TypeError
+        The `V` is not a `~astropy.units.Quantity` and cannot be
+        converted into a ~astropy.units.Quantity.
+
+    ~astropy.units.UnitConversionError
+        If the `V` is not in appropriate units.
+
+    ValueError
+        If the magnitude of `V` is faster than the speed of light.
+
+    Warns
+    -----
+    ~astropy.units.UnitsWarning
+        If units are not provided, SI units are assumed.
+
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> velocity = 1.4e8 * u.m / u.s
+    >>> mass = 1 * u.kg
+    >>> relativistic_energy(velocity, mass)
+    <Quantity 1.01638929e+17 kg m2 / s2>
+    >>> relativistic_energy(299792458*u.m/u.s, mass)
+    <Quantity inf kg m2 / s2>
+    '''
+
+    gamma = Lorentz_factor(V)
```

Comment:
It looks like you do not need to do any error handling for velocity `>= c`, since the [`Lorentz_factor` already does this](https://github.com/PlasmaPy/PlasmaPy/blob/860efdd92f766f2cbafde4b0b136bf698c9bd3d7/plasmapy/formulary/relativity.py#L68-L72).